sql_user_check_cosmos_nosql/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly CMD="az"
+readonly RESOURCE_GROUP=$RESOURCE_GROUP
+readonly ACCOUNT_NAME=$ACCOUNT_NAME
+readonly CLIENT_ID=$CLIENT_ID
```

Comment:
Any reason we have bash script ?
Instead can we have python script inside servicbility pod ?

Reply:
This repo is public, when customers prepare credentials, we ask them to run script from this repo to validate credentials before onboarding. All the scripts here are in bash.